internal/aez/ccd_power_regulators.go

```diff
@@ -34,6 +34,9 @@ func (data od) voltage() float64 {
 
 //CPRU structure
 type CPRU struct {
+	STAT uint8 // CPRU/CRB power status
```

Comment:
Do we actually want this STAT still encoded or should there be a type here `STAT CPRUStat` where `CPRUStat` implements functions for reading out `OvervolatageFaul(CCDSensor sensor) bool` and `Power(CCDSensor sensor) bool`. And `CCDSensor` is an 'enum' of masks for the sensors.